lib/cli_displayed/display_cli.py

```diff
@@ -0,0 +1,189 @@
+    >>> display_api_status_message(api_messag,1,end=False)
+    ... ├─ [x] statusCode:400
+    ... ├─ [x] message:The workId you specified was not found.
+    ... ├─ [x] requestId:xxxxxxxxxx-xx
+    """
+    # Check request status
+    if api_message["statusCode"] != 200:
+        symbol = 1
+    else:
+        symbol = 0
+    
+    # display each message except workDraft
+    count = 0
+    endl = False
+    for k,i in dict(api_message).items():
+        if k == "workDraft":
+            continue
+        count += 1
+        if end:
+            if count == len(dict(api_message)) -1: endl = True
+        display_status_symbol(order,symbol,f"{k}:{i}",endl)
+
+def display_configuration(draft,workId,ta_api,number_file):
+    """
+    Display the configuration
+
+    Parameter
+    ---------
+    draft: dict
+        outputDraft and filedraft
+    workId: int
+    ta_api: str
+        ta_api url
+    number_file:
+        number of file in Assignment directory
+    
+    Example
+    -------
+    >>> draft = {"fileDraft": "{studentId}_test.zip","outputDraft": ['studentId', 'param1', 'param2', 'comment', 'score', 'scoreTimestamp']}
+    >>> workId = 123456
+    >>> ta_api = "https://ta-api.testserver.com/"
+    >>> number_file = 50
+    >>> display_configuration(draft,workId,ta_api,number_file)
+    ... +--------------------------------------------------------------------------------------------+
+    ... |                                        Configuration                                       |
+    ... +--------------------------------------------------------------------------------------------+
+    ... |      ta-api      |                     https://ta-api.testserver.com/                      |
+    ... +--------------------------------------------------------------------------------------------+
+    ... |      workId      |                                123456                                   |
+    ... +--------------------------------------------------------------------------------------------+
+    ... |    fileDraft     |                           {studentId}_test.zip                          |
+    ... +--------------------------------------------------------------------------------------------+
+    ... |   outputDraft    | ['studentId', 'param1', 'param2', 'comment', 'score', 'scoreTimestamp'] |
+    ... +--------------------------------------------------------------------------------------------+
+    ... | Number of files  |                                   50                                    |
+    ... +--------------------------------------------------------------------------------------------+
+    
+    """
+    outputDraft = draft["outputDraft"]
+    fileDraft = draft["fileDraft"]
+    dict_ele = {"ta-api": ta_api,"workId": workId,"fileDraft": fileDraft,"outputDraft": outputDraft,"Number of files": number_file}
```

Comment:
What is this variable for? Using the full name or meaningful variable name make it easier to understand.